README.md

```diff
@@ -1,2 +1,83 @@
 [![Work in Repl.it](https://classroom.github.com/assets/work-in-replit-14baed9a392b3a25080506f3b7b6d57f295ec2978f6f33ec97e36a161684cbe9.svg)](https://classroom.github.com/online_ide?assignment_repo_id=4674825&assignment_repo_type=AssignmentRepo)
 # Careers in Computer Science
+## Description
+- Web Developer 
+- Develops the code and functionality of a website. 
+- Aims to create a website simple enough for anyone to use, but complex enough to appeal to experienced computer users as well.
+- This is in terms of the features that the website has, not the design or aesthetics of the site.
+## Responsibilities
+Web developers write the code for the user interface, web server, and database.
+User interface code allows the user to perform actions on the website. Web server code is the server code, which runs behind the website. Database code allows information to be stored and retrieved smoothly.
+## Education
+At the bare minimum, web developers really just need a high school diploma. In most jobs that demand higher qualifications, one will usually need an Associate's degree in Web Design, or a Bachelor's degree in Computer / Information Science.
+## Technical Skills
+- Knowledge of a variety of programming languages
```

Comment:
need to list specific skill.  What specific programming languages? what specific graphic design software?